src/dstack/_internal/server/services/metrics.py

```diff
@@ -131,6 +149,14 @@ def _calculate_job_metrics(points: Sequence[JobMetricsPoint]) -> JobMetrics:
     return JobMetrics(metrics=metrics)
 
 
+def _make_constant_metric(name: str, timestamps: list[datetime], value: float) -> Metric:
+    return Metric(
+        name=name,
+        timestamps=[timestamps[0], timestamps[-1]] if len(timestamps) > 1 else [timestamps[0]],
```

Comment:
I think we need to document the format for constant metrics. Also unsure if it's worth having a special case for constant metrics at all to save some bytes. I'd return all the timestamps.

Reply:
Removed the special case, constant metrics now use the same timestamps as regular metrics.